lib/http_headers.js

```diff
@@ -25,6 +24,7 @@ HttpHeaders.prototype.toJSON = function() {
     var result = {};
     forEach(this._headersByLowercasedKey, function(headerDefinition, lowercasedKey) {
         var headerKey;
+        /* istanbul ignore next */
         if (isBrowser && lowercasedKey === 'user-agent') {
```

Comment:
FYI, the reason `istanbul ignore if` doesn't work here is that there are two branches involved. There's the body of the `if` statement (which is ignored by `istanbul ignore if`) and the second half of the condition expression, `lowercasedKey === 'user-agent'` (which is *not* ignored by `istanbul ignore if`).

We could switch the order of the expressions in the conditional. That would technically be more precise, but I wouldn't push for it. The current order is presumably more intuitive for the folks maintaining this code, and it's more important to support them than it is to improve a technicality in the coverage report.